nodes/locales/en-US/clarify_insert.html

```diff
@@ -13,67 +13,22 @@ <h3>General Properties</h3>
-
-    <dt class="optional">Annotations <span class="property-type">msg.payload.annotations</span></dt>
-    <dd>Annotations</dd>
-    <dd>Expects: Map[string]string[].</dd>
-    <dd>Example: <code>{"version":["v1"]}</code></dd>
-
-    <dt class="optional">Engineering Unit <span class="property-type">msg.payload.engUnit</span></dt>
-    <dd>Engineering unit for the signal.</dd>
-    <dd>Expects: string.</dd>
-    <dd>Example: <code>bar</code></dd>
-
-    <dt class="optional">Enum values <span class="property-type">msg.payload.enumValues</span></dt>
-    <dd>Enum values for enum signals.</dd>
-    <dd>Expects: Map[number]string.</dd>
-    <dd>Example: <code>{0: "Open", 1: "Closed"}</code></dd>
-
-    <dt class="optional">Source type <span class="property-type">msg.payload.sourceType</span></dt>
-    <dd>The source type of the signal. Can be either measurement, aggregation or prediction. Defaults to measurement.</dd>
-    <dd>Expects: string.</dd>
-    <dd>Example: <code>measurement</code></dd>
-
-    <dt class="optional">Sample interval <span class="property-type">msg.payload.sampleInterval</span></dt>
-    <dd>The sample interval of the signal. Set the expected sample interval for the signal data in RFC 3339 period.</dd>
-    <dd>Expects: string as valid RFC 3339 period.</dd>
-    <dd>Example: <code>PT1S</code> (for 1 second)</dd>
-
-    <dt class="optional">Gap detection <span class="property-type">msg.payload.gapDetection</span></dt>
-    <dd>The gap detection of the signal. Set the expected gap detection time for the signal in RFC 3339 period. Common to set the gap detection 2-3 times larger than the expected sample interval.</dd>
-    <dd>Expects: string as valid RFC 3339 period.</dd>
-    <dd>Example: <code>PT3S</code> (for 3 second)</dd>
-
-    <dt>Data times<span class="property-type">msg.payload.data.times</span></dt>
-    <dd>Timestamps for the data. The length of the array must be equal to Data series</dd>
-    <dd>Expects: array</dd>
-    <dd>Example: <code>["2020-01-01T00:00:00Z",
-      "2020-01-01T00:01:00Z"]</code></dd>
-
-    <dt>Data series<span class="property-type">msg.payload.data.series</span></dt>
-    <dd>Values of the data. The length of the array must be equal to Data times</dd>
-    <dd>Expects: array</dd>
-    <dd>Example: <code>[1, 10]</code></dd>
+    <dt class="optional">payload <span class="property-type">object || undefined</span></dt>
+    <dd>
+      Object containing two elements; times and values.
+      <code>times: ["&lt;timestamp&gt;", ...]</code>
+      <code>values: [(&lt;number&gt; || null), ...]</code>
+      If omitted will no data metrics be sent to Clarify.
+    </dd>
+
+    <dt class="optional">signal <span class="property-type">object&lt;Signal&gt; || undefined</span></dt>
+    <dd>
+      Meta data for the signal. Matches <a href="https://docs.clarify.us/reference#signal">signal object in Clarify documentation</a>.
+      If omitted will no meta data be changed in Clarify.
+    </dd>
```

Comment:
I guess there is no way to view the rendered documentation here?

Reply:
It's quite easy. Clone [searis/node-red-dev-env](https://github.com/searis/node-red-dev-env), and this repo in an adjacent folder. While in the `node-red-dev-env` folder, run:
```sh
$ npm run watch
```
and access `http://localhost:1880/node-red`.

Is documented in README.md at [searis/node-red-dev-env](https://github.com/searis/node-red-dev-env).